alerter.c

```diff
@@ -27,6 +38,7 @@ int main() {
     alertInCelcius(400.5);
     alertInCelcius(303.6);
     printf("%d alerts failed.\n", alertFailureCount);
+    assert(alertFailureCount == 1);

```

Comment:
min of 3 values need to be tested i.e. < threshold, == threshold and > threshold ...